### PR TITLE
Add support for Miniscope-DAQ-V3 and MiniscopeAnalysis.  Update readme and background.

### PR DESCRIPTION
**Pull request by kabilar**

### CODE REVIEW

File: element_miniscope/imaging.py [hunk 1]
```diff
@@ -694,6 +740,27 @@ def make(self, key):
...
+            # infer "segmentation_channel" - from params if available, else from miniscope analysis loader
+            params = (ProcessingParamSet * ProcessingTask & key).fetch1('params')
+            seg_channel = params.get('segmentation_channel', loaded_miniscope_analysis.segmentation_channel)
+
+            fluo_traces = []
+            for mask in loaded_miniscope_analysis.masks:
+                fluo_traces.append({**key, 
+                                    'mask': mask['mask_id'], 
+                                    'fluo_channel': seg_channel,
+                                    'fluorescence': mask['raw_trace']})
```
Comment (dimitri-yatsenko) [inline element_miniscope/imaging.py:759]:
```suggestion
            fluo_traces = [dict(key,
                                mask=mask['mask_id'],
                                fluo_channel=seg_channel,
                                fluorescence=mask['raw_trace'])
                           for mask in loaded_miniscope_analysis.masks]
```

File: element_miniscope/imaging.py [hunk 2]
```diff
@@ -772,6 +839,28 @@ def make(self, key):
...
+                # infer "segmentation_channel" - from params if available, else from miniscope analysis loader
+                params = (ProcessingParamSet * ProcessingTask & key).fetch1('params')
+                seg_channel = params.get('segmentation_channel', loaded_miniscope_analysis.segmentation_channel)
+
+                activities = []
+                for mask in loaded_miniscope_analysis.masks:
+                    activities.append({**key, 
+                                       'mask': mask['mask_id'],
+                                       'fluo_channel': seg_channel,
+                                       'activity_trace': mask[attr_mapper[key['extraction_method']]]})
```
Comment (dimitri-yatsenko) [inline element_miniscope/imaging.py:860]: Replace with list apprehension.

File: element_miniscope/imaging.py [hunk 3]
```diff
@@ -83,12 +84,14 @@ class ProcessingParamSet(dj.Lookup):
     param_set_hash: uuid
     unique index (param_set_hash)
     params: longblob  # dictionary of all applicable parameters
+    package_version=null: varchar(16)
```
Comment (ttngu207) [inline element_miniscope/imaging.py:87]: I recall we discussed having `package_version` as secondary attribute at `Processing` table

File: element_miniscope/imaging.py [hunk 4]
```diff
@@ -83,12 +84,14 @@ class ProcessingParamSet(dj.Lookup):
...
     """
 
     @classmethod
     def insert_new_params(cls, processing_method: str, paramset_idx: int,
                           paramset_desc: str, params: dict):
         param_dict = {'processing_method': processing_method,
+                      'package_version': package_version,
```
Comment (ttngu207) [inline element_miniscope/imaging.py:94]: same as the comment above, `package_version` at `Processing` table

**Review by ttngu207 — CHANGES_REQUESTED** [review]
As this is now a new Element specific to Miniscope, we can clean up the design a bit:
+ remove any z-plane?
+ no need for the Field table (as there will be a single field)?
+ remove suite2p, scanimage, scanbox loader

**Comment by kabilar** [comment]
Will address comments and add remaining changes in future pull request.

**Pull request merged**